packages/iTwinUI-react/src/core/Header/HeaderDropdownButton.tsx

```diff
@@ -5,11 +5,9 @@
 import cx from 'classnames';
 import React from 'react';
 import { DropdownMenu } from '../DropdownMenu';
-import SvgCaretDownSmall from '@itwin/itwinui-icons-react/cjs/icons/CaretDownSmall';
-import SvgCaretUpSmall from '@itwin/itwinui-icons-react/cjs/icons/CaretUpSmall';
 import { PolymorphicForwardRefComponent, useTheme } from '../utils';
 import { DropdownButtonProps } from '../Buttons';
-import { useMergedRefs } from '../utils';
+import { useMergedRefs, SvgCaretDownSmall, SvgCaretUpSmall } from '../utils';
```

Comment:
These imports can be combined with the `PolymorphicForwardRefComponent` and `useTheme` imports. I'm not sure why they were separated in the first place

Reply:
Done